pbft/pbft.cpp

```diff
@@ -1887,3 +1888,20 @@ uint32_t pbft::generate_random_number(uint32_t min, uint32_t max)
     std::uniform_int_distribution<uint32_t> dist(min, max);
     return dist(gen);
 }
+
+void pbft::add_session_to_pool(const std::string& msg_hash, std::shared_ptr<bzn::session_base> session)
```

Comment:
This name feels strange to me; it's not a pool of interchangeable sessions

Reply:
renamed to add_session_to_sessions_waiting